devenv/loadtest/run.sh

```diff
@@ -22,7 +26,17 @@ run() {
 	done
 	shift $((OPTIND-1))
 
-  docker run -t --network=host -v $PWD:/src -e URL=$url --rm -i loadimpact/k6:master run --vus $vus --duration $duration src/render_test.js
+  docker run \
+    -it \
+    --network=host \
+    --mount type=bind,source=$PWD,destination=/src \
+    -e URL=$url \
+    --rm \
+    loadimpact/k6:master run \
+    --vus $vus \
+    --duration $duration \
+    $iterationsOption \
```

Comment:
```suggestion
    --iterations $iterations \
```

Reply:
I think this would cause the script to fail if `$iterations` is not specified. And you want the ability to leave the option out entirely, so you can do things like "run as many iterations as possible over 5 minutes".

Reply:
Ah yes you're right! OK, then, I guess you can merge 🙂